Makefile

```diff
@@ -0,0 +1,54 @@
+# Copyright (2019) Cobalt Speech and Language Inc.
+
+.PHONY: all deps gen depsbin deps-protoc
+all: gen
+
+SHELL := /bin/bash
+
+TOP := $(shell pwd)
+
+DEPSBIN := ${TOP}/deps/bin
+DEPSGO := ${TOP}/deps/go
+DEPSTMP := ${TOP}/deps/tmp
+$(shell mkdir -p $(DEPSBIN) $(DEPSGO) $(DEPSTMP))
+
+DEPSVENV := ${TOP}/deps/venv
+
+export PATH := ${DEPSBIN}:${DEPSGO}/bin:$(PATH)
+deps: deps-protoc deps-hugo deps-gendoc deps-gengo deps-gengateway deps-py
+
+deps-protoc: ${DEPSBIN}/protoc
+${DEPSBIN}/protoc:
+	cd ${DEPSBIN}/../ && wget \
+		"https://github.com/protocolbuffers/protobuf/releases/download/v3.7.1/protoc-3.7.1-linux-x86_64.zip" && \
+		unzip protoc-3.7.1-linux-x86_64.zip && rm -f protoc-3.7.1-linux-x86_64.zip
```

Comment:
This will only work on linux, not macOS. We should document that--perhaps even have a top-level check that errors out immediately if it's run on OSX.  Otherwise the errors will come later in the process and be less obvious.

Reply:
I'll document it in the readme for now and we can address this for OSX later. The binaries are available for OSX too, so it shouldn't be a problem.